server/app/assets/javascripts/trusted_intermediary.ts

```diff
@@ -0,0 +1,25 @@
+// Javascript handling for the trusted intermediary page
+
+import {addEventListenerToElements} from './util'
+
+export function init() {
+  updateListeners()
+}
+
+/** Safe to call multiple times */
+export function updateListeners() {
+  addEventListenerToElements('#cf-ti-clear-search', 'click', resetSearchForm)
+}
+
+/** In the enumerator form - add a new input field for a repeated entity. */
```

Comment:
suggestion (blocking): update this comment (I'm thinking it was copied over?)

Reply:
Thanks for catching that! I copied and pasted and then forgot to look at the whole class again. Made a few related cleanups.